core/src/main/resources/reference.conf

```diff
@@ -206,183 +77,188 @@ cassandra-journal {
+    # this can be relaxed by setting this property to off.
+    gap-free-sequence-numbers = on
+
+    # When using LQ writing in one DC and querying in another, the events for an entity may
+    # appear in the querying DC out of order, when that happens, try for this amount of
+    # time to find the in-order sequence number before failing the stream
+    events-by-persistence-id-gap-timeout = 10s
+
+    # How many events to fetch in one query (replay) and keep buffered until they
+    # are delivered downstreams.
+    max-buffer-size = 500
+
+    # Configure this to the first bucket eventByTag queries will start from in the format
+    # yyyyMMddTHH:mm yyyyMMdd is also supported if using Day as a bucket size
+    # Will be rounded down to the start of whatever time bucket it falls into
+    # When NoOffset is used it will look for events from this day and forward.
+    first-time-bucket = "20151120T00:00"
+
+    # Deserialization of events is perfomed in an Akka streams mapAsync operator and this is the
+    # parallelism for that. Increasing to means that deserialization is pipelined, which can
+    # be an advantage for machines with many CPU cores but otherwise it might be slower because
+    # of higher CPU saturation and more competing tasks when there are many concurrent queries or
+    # replays.
+    deserialization-parallelism = 1
 
-  # The time to wait before cassandra will remove the tombstones created for deleted entries.
-  # cfr. gc_grace_seconds table property documentation on 
-  # http://www.datastax.com/documentation/cql/3.1/cql/cql_reference/tabProp.html
-  gc-grace-seconds = 864000
-
-  # Number load attempts when recovering from the latest snapshot fails
-  # yet older snapshot files are available. Each recovery attempt will try
-  # to recover using an older than previously failed-on snapshot file 
-  # (if any are present). If all attempts fail the recovery will fail and
-  # the persistent actor will be stopped.
-  max-load-attempts = 3
-
-  # Enable Dropwizard Metrics for Cluster.
-  # See: https://docs.datastax.com/en/developer/java-driver/3.5/manual/metrics/
-  metrics-enabled = on
-
-  # Enable JMX reporter for Dropwizard Metrics.
-  # See: https://docs.datastax.com/en/developer/java-driver/3.5/manual/metrics/
-  jmx-reporting-enabled = on
-}
+  }
 
-# This configures the default settings for all CassandraReadJournal plugin
-# instances in the system.
-#
-# If you use multiple plugin instances you need to create differently named 
-# sections containing only those settings that shall be different from the defaults
-# configured here, importing the defaults like so:
-#
-#   my-cassandra-query-journal = ${cassandra-query-journal}
-#   my-cassandra-query-journal {
-#     <settings...>
-#   }
-cassandra-query-journal {
-  # Implementation class of the Cassandra ReadJournalProvider
-  class = "akka.persistence.cassandra.query.CassandraReadJournalProvider"
+  events-by-tag {
```

Comment:
placed everything for `events-by-tag` in this section. Alternatively it could be split into write and read sections, but I think that is more complicated for end-users. wdyt?

Reply:
This looks good